.github/workflows/update-changelog.yaml

```diff
@@ -0,0 +1,32 @@
+name: "Update changelog"
+
+on:
+  workflow_dispatch: {}
+  release:
+    types: [published, edited, released, unpublished, deleted]
+jobs:
+  upgrade-kuma:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: kong-mesh
+    steps:
+      - name: "Clone Kuma"
+        uses: actions/checkout@v2
+
```

Comment:
Maybe use the github app? I just added the secrets
https://github.com/kumahq/kuma-website/blob/master/.github/workflows/sync-generated.yml#L10-L15
https://github.com/kumahq/kuma-website/blob/master/.github/workflows/sync-generated.yml#L33

Reply:
makes snese!